src/test/scala/org/polystat/py2eo/Commons.scala

```diff
@@ -0,0 +1,50 @@
+package org.polystat.py2eo
+
+import org.junit.Assert.assertTrue
+import org.polystat.py2eo.Main.{debugPrinter, writeFile}
+
+import java.io.File
+import scala.sys.process._
+
+trait Commons {
+  val testsPrefix: String = System.getProperty("user.dir") + "/src/test/resources/org/polystat/py2eo/"
```

Comment:
Why not loading resourses from classpath?

Reply:
resolved